### PR TITLE
update all veba core-components to a higher version

### PR DESCRIPTION
**Pull request by rguske**
## Summary

This PR will almost push every in the veba-bom.json listed component of VEBAs architecture to a higher version. This includes components like Kubernetes, Knative, Antrea, Cert-Manager, Contour, etc.

Please describe your changes and what you plan to introduce as part of this pull request submission. If it fixes a bug or resolves a feature request, be sure to link to the issue numbers following the template below.

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [x] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [x] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [x] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commits related to your change
- [x] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [ ] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation changes
- [x] Other (please describe):

Version updates.

## Resolved Issues

List of Issues closed or resolved by this PR. Add multiple `Closes` keyword followed by the issue number (e.g. Closes 

Closes: #1158

## Testing Verification

* built appliance locally
* made sure VEBA can be deployed in an air-gapped environment
* validated function deployment
* validated VEBA UI plugin works
* checked all endpoints
* deployed VEBA multiple times

### CODE REVIEW

File: veba-bom.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "veba": {
-        "version": "v0.7.5"
+        "version": "v0.8.0"
```
Comment (lamw) [inline veba-bom.json:3]: Please revert back to `v0.7.5` as the VEBA version changes only occurs when we do a release

File: veba-bom.json [hunk 2]
```diff
@@ -17,136 +17,138 @@
...
         "gitRepoTag": "main",
         "containers": [{
             "name": "kindest/node",
-            "version": "v1.25.3"
+            "version": "v1.29.1"
         }]
     },
     "kubernetes": {
-        "gitRepoTag": "v1.25.5",
+        "packageRepoTag": "v1.29",
```
Comment (lamw) [inline veba-bom.json:53]: Let's not introduce a new key here, given the difference between `packageRepoTag` and `gitRepoTag` is just removing the last octet, lets do that in shell script itself

File: scripts/photon-settings.sh [hunk 3]
```diff
@@ -36,16 +36,15 @@ tdnf install -y \
   httpd
 
 echo '> Adding K8s Repo'
-curl -L https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg -o /etc/pki/rpm-gpg/GOOGLE-RPM-GPG-KEY
-rpm --import /etc/pki/rpm-gpg/GOOGLE-RPM-GPG-KEY
+K8S_PACKAGE_REPO_VERSION=$(jq -r < ${VEBA_BOM_FILE} '.["kubernetes"].packageRepoTag')
```
Comment (lamw) [inline scripts/photon-settings.sh:39]: Replace this with following (which uses string expansion to strip away anything from the last period)

```
K8S_PACKAGE_REPO_VERSION=${$(jq -r < ~/git/vcenter-event-broker-appliance/veba-bom.json '.["kubernetes"].gitRepoTag')%.*}
```

File: veba-bom.json (hunk 1 above)
Comment (rguske) [reply veba-bom.json]: Done

File: veba-bom.json (hunk 2 above)
Comment (rguske) [reply veba-bom.json]: Done

File: scripts/photon-settings.sh (hunk 3 above)
Comment (rguske) [reply scripts/photon-settings.sh]: It's now:
```shell
K8S_PACKAGE_REPO_VERSION_FULL=$(jq -r < "${VEBA_BOM_FILE}" '.["kubernetes"].gitRepoTag')
K8S_PACKAGE_REPO_VERSION=${K8S_PACKAGE_REPO_VERSION_FULL%.*}
```

**Review by lamw — APPROVED** [review]
🔥 LGTM!

**Pull request merged**